test/e2e/common/container_probe.go

```diff
@@ -211,7 +212,7 @@ var _ = framework.KubeDescribe("Probing container", func() {
 		Testname: Pod liveness probe, docker exec, restart
 		Description: A Pod is created with liveness probe with a Exec action on the Pod. If the liveness probe call  does not return within the timeout specified, liveness probe MUST restart the Pod.
 	*/
-	ginkgo.It("should be restarted with a docker exec liveness probe with timeout ", func() {
+	ginkgo.It("should be restarted with an exec liveness probe with timeout [NodeConformance]", func() {
```

Comment:
I added this temporarily so that `pull-kubernetes-e2e-node` (running dockershim) would start running this, since that job only runs `[NodeConformance]` tests. This was why this failing test was not caught in https://github.com/kubernetes/kubernetes/pull/94115, since only jobs running containerd ran the full e2e suite. Correct me if I got anything incorrect here @hasheddan. 

I think it's appropriate to add this test to `[NodeConformance]` but I'll defer to SIG Node leads if we're okay with this now or if we should defer this discussion to a separate PR

Reply:
@andrewsykim this is mostly correct :+1: I believe right now we actually are split between some of the e2e suite being run against dockershim and some being run against containerd. As @dims pointed out above, it would be a good first step to get an optional test that we can invoke as a presubmit that will run the full test suite against either.

I think you are correct that we need to defer adding `[NodeConformance]` to this test to a separate PR.

Reply:
i agree we should defer labeling this NodeConformance

Reply:
ack, I'll remove this

Reply:
Removed, thanks